types/pnpapi/index.d.ts

```diff
@@ -39,3 +40,11 @@ export function resolveRequest(
 ): string | null;
 
 export function setup(): void;
+
+declare global {
+	namespace NodeJS {
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/pnpapi/index.d.ts;
  Line: 45;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.